src/components/styles/Theme/ThemeProvider.tsx

```diff
@@ -150,7 +150,7 @@ interface LinkTheme {
   disableTargetIcon: boolean;
 }
 
-interface NavbarTheme {
+export interface NavbarTheme {
```

Comment:
Product apps have been mirroring this, we should rely on the types exported by the theme provider instead going forwards.